treeherder/etl/classification_mirroring.py

```diff
@@ -74,7 +75,7 @@ def send_request(self):
         es_url = "".join([es_host, es_endpoint])
         logger.info("Sending data to %s: %s", es_url, self.body)
         headers = {'Content-Type': 'text/plain', 'Connection': 'close'}
-        r = requests.post(es_url, data=json.dumps(self.body), headers=headers)
+        r = requests.post(es_url, data=json.dumps(self.body), headers=headers, timeout=self.timeout)
```

Comment:
Let's use settings.TREEHERDER_REQUESTS_TIMEOUT directly here, since it's a constant and doesn't vary by object :-)